### DISCUSSION TITLE
Saucedacity needs a saucy icon, or does it?

### DISCUSSION THREAD

**Discussion by ghost** [saucedacity/saucedacity#78] [Ideas]
Saucedacity has a disaster: When your device is set to dark mode, it's almost impossible to locate the icon, especially when I've got a butt ton of apps installed.
I'm skeptical over the fact that _will it get a new icon_ or _is it going to use the Tenacity icon once the projects merge_?

**Comment by generic-pers0n** [comment]
> Saucedacity has a disaster: When your device is set to dark mode, it's almost impossible to locate the icon, especially when I've got a butt ton of apps installed.
> I'm skeptical over the fact that _will it get a new icon_ or _is it going to use the Tenacity icon once the projects merge_?

When the projects merge, we will use all Tenacity branding. This includes using the Tenacity icon.
